tournaments/index.js

```diff
@@ -794,6 +794,7 @@ var commands = {
 			if (params.length < 1) {
 				return this.sendReply("Usage: " + cmd + " <minutes|off>");
 			}
+			if (params[0] === Infinity || params[0] === 0) params[0] = 'off';
```

Comment:
I'm pretty sure this will literally never happen, since `params[0]` is always a string.

(I'm saying this with zero familiarity with this code because if `params[0]` were ever not a string, `params[0].toLowerCase()` would crash.